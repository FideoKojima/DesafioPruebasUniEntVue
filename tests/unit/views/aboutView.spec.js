import { mount } from '@vue/test-utils';
import About from '@/views/AboutView.vue';

describe('About.vue', () => {
    test('Vista renderiza correctamente', () => {
        const wrapper = mount(About);
        expect(wrapper.exists()).toBe(true); 
    });
    
    test('coincide snapshot', () => {
        const wrapper = mount(About);
        expect(wrapper.html()).toMatchSnapshot(); 
    });
});
