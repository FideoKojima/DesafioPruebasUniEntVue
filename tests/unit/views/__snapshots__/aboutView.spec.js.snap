// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue coincide snapshot 1`] = `
<div>
  <h1>PÃ¡gina About</h1>
</div>
`;
